tests/power_cycle/test_powercycle_tools.py

```diff
@@ -0,0 +1,153 @@
+from pprint import pformat
+
+import matplotlib.pyplot as plt
+import pytest
+
+from bluemira.base.look_and_feel import bluemira_debug
+from bluemira.power_cycle.tools import (
+    _add_dict_entries,
+    _join_delimited_values,
+    adjust_2d_graph_ranges,
+    validate_axes,
+)
+
+
+def script_title():
+    return "Test Power Cycle 'tools'"
+
+
+def test_add_dict_entries():
+    old_dict = {
+        "value_1": 1,
+        "value_2": 2,
+        "value_3": 3,
+    }
+    new_entries = {
+        "value_1": 10,
+        "value_4": 40,
+    }
+    new_dict = _add_dict_entries(old_dict, new_entries)
+    bluemira_debug(
+        f"""
+        {script_title()} (_add_dict_entries)
+
+        Old dictionary:
+        {pformat(old_dict)}
+
+        New dictionary entries:
+        {pformat(new_entries)}
+
+        New dictionary:
+        {pformat(new_dict)}
+        """
+    )
+    assert isinstance(new_dict, dict)
+    assert new_dict == {**old_dict, **new_entries}
+    assert 0
+
+
+def test_join_delimited_values():
+    all_arguments = [
+        ["value 1", "value 2", "value 3"],
+        {"value_1": 1, "value_2": 2, "value_3": 3},
+    ]
+    for test_argument in all_arguments:
+        string_from_argument = _join_delimited_values(test_argument)
+        bluemira_debug(
+            f"""
+            {script_title()} (_join_delimited_values)
+
+            Generated string:
+            {string_from_argument}
+            """
+        )
+    assert isinstance(string_from_argument, str)
```

Comment:
This test is just testing builtin python functionality both of the tests here can be accomplished by 
`' ,'.join(all_arguments[0])` or `' ,'.join(all_arguments[1])`. Remove both the function and the test.

```suggestion
def test_join_delimited_values():
    all_arguments = [
        ["value 1", "value 2", "value 3"],
        {"value_1": 1, "value_2": 2, "value_3": 3},
    ]
    for test_argument in all_arguments:
        string_from_argument = _join_delimited_values(test_argument)
        bluemira_debug(
            f"""
            {script_title()} (_join_delimited_values)

            Generated string:
            {string_from_argument}
            """
        )
    assert isinstance(string_from_argument, str)
```